.github/workflows/publish.yml

```diff
@@ -0,0 +1,43 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+
+jobs:
+  check:
+    name: Check commit message
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Check commit message
+        uses: adityaa30/check-commit@master
+        with:
+          max-header-length: 255
+
+  release:
+    needs: test
+    if: github.ref == 'refs/heads/master'
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Publish
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
```

Comment:
@kto knows how to create a PyPi project and get the token which should be added to the [repo secrets](https://github.com/F-Secure/fliprate_actions/settings/secrets/actions)